README with settings info (#619)
wtsi-npg/npg_tracking: README.md

+ `NPG_WEBSERVICE_CACHE_DIR` - a cache directory for test LIMS and NPG servers
                               XML feeds, used in unit tests only

Tags and their Meaning
----------------------

Text tags can be associated with sequencing runs and individual lanes of a run.
Arbitrary tags are supported. Tags can be assigned both manually by the users
via a web page of a run and automatically by differnt cron jobs. Access to the
tracking database is needed for the latter.

Formally speaking, the tags are not curated, but some tags trigger special
features in the data processing pipelines. Some of special tags are listed
below. Once associated with the run, the tag is not removed, unless this is
explicitly stated. Any tag can be removed manually via a web page of a run.

+ `staging` - A run is assigned this tag the first time the run folder is seen
              by the staging daemon, see the `staging_area_monitor` script in
              this package. The tag is removed by a cron job that deletes
              run folders from the staging area.
+ `multiplex` - The staging daemon assigns this tag to a run if the run had
                an indexing read.

> + `no_mqc_skipper` - This tag can be assigned to a run to prevent the data

Reviewer: ```suggestion
+ `no_mqc_skipper` - This tag can be manually assigned to a run to prevent the data
```

> + `no_mqc_skipper` - This tag can be assigned to a run to prevent the data
>                      being assessed by a script that can change the run status
>                      from `qc review pending` to `archival pending` bypassing
>                      the stage of manual QC.

Reviewer: ```suggestion
+ `no_mqc_skipper` - This tag can be manually assigned to a run to
                     prevent the data being assessed by a script that can
                     change the run status from `qc review pending` to 
                     `archival pending` bypassing the stage of manual QC.
```

> + `no_auto` - This tag can be assigned to a run to prevent it being considered
>               by any automatic processing. Currently, only functionaly
>               identical to triggered by the `no_mqc_skipper` is implemented.

Reviewer: ```suggestion
+ `no_auto` - This tag is intended to be manually assigned to a run to prevent
              it being considered by any automatic processing. Currently, it is only
              functionally identical to the `no_mqc_skipper` tag.
```
